scripts/minikube.sh

```diff
@@ -95,6 +95,18 @@ function install_kubectl() {
     curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/"${KUBE_VERSION}"/bin/linux/"${MINIKUBE_ARCH}"/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/
 }
 
+function validate_container_cmd() {
+    if [[ "${CONTAINER_CMD}" == "docker" ]] || [[ "${CONTAINER_CMD}" == "podman" ]]; then
+        if ! command -v "${CONTAINER_CMD}" &> /dev/null; then
+            echo "'${CONTAINER_CMD}' not found"
+            exit 1
+        fi
+    else
+        echo "'CONTAINER_CMD' should be either docker or podman and not '${CONTAINER_CMD}'"
```

Comment:
Are the single quotes on `${CONTAINER_CMD}` required? Do we wish to execute that?

Reply:
There are two CONTAINER_CMD in the line.
First one will not be executed and the second one is to access the variable to show what user had passed in.

Example:
```
[0] pkalever 😎 ceph-csi✨ CONTAINER_CMD="dockerx" ./scripts/minikube.sh  cephcsi 
copying the cephcsi image                                                         
'CONTAINER_CMD' should be either docker or podman and not 'dockerx'               
[1] pkalever 👿 ceph-csi✨                                                        
```

Reply:
Thanks, the output looks clearer in this approach.